authentication/account/service/user.go

```diff
@@ -145,8 +145,27 @@ func (s *userServiceImpl) DeactivateUser(ctx context.Context, username string) (
 		}
 
 		// revoke all user's tokens
-		return s.Services().TokenService().SetStatusForAllIdentityTokens(ctx, identity.ID, token.TOKEN_STATUS_REVOKED)
+		err = s.Services().TokenService().SetStatusForAllIdentityTokens(ctx, identity.ID, token.TOKEN_STATUS_REVOKED)
+		if err != nil {
+			return err
+		}
+
+		return nil
 	})
+	if err != nil {
+		return nil, err
+	}
+
+	// call WIT and Tenant to deactivate the user there as well,
+	// using `auth` SA token here, not the request context's token
+	err = s.Services().WITService().DeleteUser(ctx, identity.ID.String())
+	if err != nil {
+		return nil, err
```

Comment:
We are about to remote WIT :) Let's not make hard dependencies to it. Just log the error and proceed maybe?

Reply:
ok, let me just log the errors but not block. Maybe even for `tenant` BTW.

Reply:
fixed in c7d24bd

Reply:
No. Let’s keep tenant call intact. If we failed to delete namespaces we actually failed to deactivate.